### ISSUE TITLE
Major performance regression with panning

### ISSUE THREAD

**Issue by alexarice** [sd-visualiser/sd-visualiser#71]
Panning ad.chil is really slow atm

**Comment by alexarice** [comment]
```
4ad0a023cc4c7247ff50841b0bf6d5fe8a1a765e is the first bad commit
commit 4ad0a023cc4c7247ff50841b0bf6d5fe8a1a765e
Author: Calin Tataru <calintat@gmail.com>
Date:   Wed Aug 30 18:16:01 2023 +0100

    Allow cutting wires in the UI (finally)

 sd-core/src/graph.rs          | 18 ++++++++++++-
 sd-core/src/hypergraph/cut.rs | 10 +++++++
 sd-core/src/interactive.rs    | 11 +++++---
 sd-graphics/src/render.rs     | 15 ++++++-----
 sd-graphics/src/renderable.rs | 61 ++++++++++++++++++++++++++++++++++++++++---
 sd-graphics/src/shape.rs      | 12 ++++++++-
 sd-gui/src/graph_ui.rs        | 10 ++++---
 sd-gui/src/selection.rs       |  6 ++---
 8 files changed, 122 insertions(+), 21 deletions(-)
```

**Comment by calintat** [comment]
Fixed by 2d93daf7398295ffd15e7b652277735b66b67c7c and 17272613c327604a757f8f22119a6de55debb6e5
